src/olympia/search/tests/test_search_ranking.py

```diff
@@ -97,10 +98,10 @@ def setUpTestData(cls):
             name='Tab Center Redux',
             slug='tab-center-redux',
             summary='Move your tabs to the side of your browser window.',
-            weekly_downloads=915,
+            weekly_downloads=305,
         )
         amo.tests.addon_factory(
-            average_daily_users=468126,
+            average_daily_users=468127,
```

Comment:
These `average_daily_users` and `weekly_downloads` changes are unexpected - are the search results excessively wrong now? (Presumably searching for "download" and "down them all" exposed the problem?)

Reply:
Just an overzealous search&replace on my part. I'll revert them, I don't think it should matter.